### ISSUE TITLE
Update version 0.6.0 "!heal" issue

### ISSUE THREAD

**Issue by khidden54** [mircearoata/SatisfactoryPakUtilityMod#26]
Seams to be some issue with the !heal cmd from latest update. Any time it is used, the game freezes and I end up having to force close the game through task manager.

**Comment by mircearoata** [comment]
Please send the game log (you can use the debug info button from SMM). I don't get a crash on any of the commands in either single player or multiplayer.

**Comment by edgarcnp** [comment]
[SMMDebug_2022-7-21_18-59-57.zip](https://github.com/mircearoata/SatisfactoryPakUtilityMod/files/9162005/SMMDebug_2022-7-21_18-59-57.zip)

**Comment by mircearoata** [comment]
From your log and another log on discord I believe it only happens on Early Access and not on Experimental, which is what I tested on, but now I tried Early Access too and got the freeze.
The common thing between the two commands is healing the player, though I haven't changed anything in the code for those.
I think I will just make the latest version be only for Update 6, because I can't find anything different between the previous mod version and this one that could cause it.
